### ISSUE TITLE
hvrbox News display has issues with keyboard navigation

### ISSUE THREAD

**Issue by robgasc** [ATLAS-Illinois/Illinois-Framework-Theme#1120]
**Describe the bug**
When tabbing through the hoverbox, the content hidden is not visible when keyboard tabs into the hoverbox.

**To Reproduce**
1. Tab through the page into a hoverbox.
2. See the slide happen but no text appear.

**Expected behavior**
When tabbed onto, the hoverbox should open and show the text.

*** LAS Issue ***
https://airtable.com/appRAPJCKVmGn7rwA/tblE7cPZhYPK9qvet/viwDBCVt46hi17o8t/recgJyUC6qwdVlV2t?blocks=hide

**Comment by trubach** [comment]
The changes in PRs #1121 and #1122 change the behavior of the news hoverbox from a "clickable card" to making it where you have to click on the title of the news story to open the article. I'd like to discuss the behavior change in our mid-sprint meeting today and have us decide as a group what the best implementation would be. I'll capture what the group decides in a follow up comment.

**Comment by robgasc** [comment]
Past Robbie was silly. I'll work on making the changes more consistent with the current styling.

**Comment by trubach** [comment]
This is fixed in the 4.x branch (PR #1121), but needs work in the 5.x branch (PR #1122).

In the 5.x branch there is a default focus CSS style that applies a blue background that's covering up the box. We'll need to update the CSS so that the text is still visible. Example below:

<img width="1339" height="637" alt="Image" src="https://github.com/user-attachments/assets/f0edeff8-5d92-45ea-89f3-96057a14ec27" />

**Comment by robgasc** [comment]
That issue appears to be pre-existing the work I've done in my branch. But I'll take a quick swing at it to see what comes out.
